lang/sr.json

```diff
@@ -16,202 +16,202 @@
+		"115": "Датотека или фасцикла са истим називом већ постоји.",
+		"116": "Фасцикла није пронађена. Освежите страницу и покушајте поново.",
+		"117": "Датотека није пронађена. Освежите листу датотека и покушајте поново.",
 		"118": "Путања извора и одредишта су исте.",
-		"201": "Датотека с истим називом већ постоји. Отпремљена датотека је промјењена у \"%1\".",
+		"201": "Датотека са истим називом већ постоји. Учитана датотека је промењена у \"%1\".",
 		"202": "Неисправна датотека.",
 		"203": "Неисправна датотека. Величина датотеке је превелика.",
-		"204": "Отпремљена датотека је неисправна.",
-		"205": "Не постоји привремена фасцикла за отпремање на серверe.",
+		"204": "Учитана датотека је неисправна.",
+		"205": "Не постоји привремена фасцикла за учитавање на серверe.",
 		"206": "Слање је поништено због сигурносних поставки. Назив датотеке садржи HTML податке.",
-		"207": "Отпремљена датотека је промјењена у \"%1\".",
-		"300": "Премјештање датотеке(а) није успјело.",
-		"301": "Копирање датотеке(а) није успјело.",
-		"302": "Deleting file(s) failed.",
-		"500": "Претраживање датотека није дозвољено из сигурносних разлога. Молимо контактирајте администратора система како би провјерили поставке CKFinder конфигурационе датотеке.",
-		"501": "Thumbnail подршка није омогућена."
+		"207": "Учитана датотека је промењена у \"%1\".",
+		"300": "Премештање датотеке(а) није успело.",
+		"301": "Копирање датотеке(а) није успело.",
+		"302": "Брисање датотеке(а) није успело.",
+		"500": "Претраживање датотека није дозвољено из сигурносних разлога. Молимо контактирајте администратора система како би проверили поставке CKFinder конфигурационе датотеке.",
+		"501": "Подршка за приказ није омогућена."
 	},
 	"ErrorMsg": {
-		"FileInvChar": "Назив датотеке не смије садржавати нити један од сљедећих знакова: \n\\ / : * ? \" < > |",
-		"FolderInvChar": "Назив фасцикле не смије садржавати нити један од сљедећих знакова: \n\\ / : * ? \" < > |"
+		"FileInvChar": "Назив датотеке не сме садржавати ни један од следећих знакова: \n\\ / : * ? \" < > |",
+		"FolderInvChar": "Назив фасцикле не сме садржавати ни један од следећих знакова: \n\\ / : * ? \" < > |"
 	},
 	"Fileeditor": {
-		"contextMenuName": "Промјени"
+		"contextMenuName": "Промени"
 	},
 	"Maximize": {
 		"maximize": "Повећај",
 		"minimize": "Смањи"
 	},
 	"Filter": {
-		"filterPlaceholder": "Filter"
+		"filterPlaceholder": "Филтер"
 	},
 	"EditImage": {
-		"apply": "Apply",
-		"adjust": "Adjust",
-		"crop": "Crop",
-		"presets": "Presets",
-		"resize": "Resize",
-		"rotate": "Rotate",
-		"keepAspectRatio": "Keep Aspect Ratio",
-		"rotateClockwise": "90&deg; Right",
-		"rotateAntiClockwise": "90&deg; Left",
+		"apply": "Примени",
+		"adjust": "Прилагоди",
+		"crop": "Обрежи",
+		"presets": "Унапред постављен",
+		"resize": "Промена величине",
+		"rotate": "Ротирај",
+		"keepAspectRatio": "Задржи омер",
+		"rotateClockwise": "90&степени; Десно",
```

Comment:
`&deg;` is an entity for the "degree" character (&deg;) and should not be replaced.